source/features/index.tsx

```diff
@@ -211,12 +211,15 @@ function enforceDefaults(
 	}
 }
 
+const getFeatureID = (url: string): FeatureID => url.split('/').pop()!.split('.', 1)[0];
```

Comment:
I think regex would be more readable. WDYT?

Reply:
> I think regex would be more readable. WDYT?

I don't think so. This seems simple enough to me